src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String START_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> arrayOfWords = new ArrayList<>();
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
```

Comment:
```suggestion
        try  (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName)));
```
it's a bad practic to leave connection to file opened